netbox/dcim/models/device_components.py

```diff
@@ -725,6 +725,13 @@ class Interface(ModularComponentModel, BaseInterface, CabledObjectModel, PathEnd
         object_id_field='assigned_object_id',
         related_query_name='interface',
     )
+    vlan_translation_policy = models.ForeignKey(
+        to='ipam.VLANTranslationPolicy',
+        on_delete=models.SET_NULL,
```

Comment:
We should prevent the deletion of a policy that is in use.
```suggestion
        on_delete=models.PROTECT,
```